.github/workflows/test.yml

```diff
@@ -28,18 +74,27 @@ jobs:
       with:
         python-version: ${{ matrix.cfg.py }}
 
+    - name: Download the wheel
+      uses: actions/download-artifact@v4
+      with:
+        name: wheel
+        path: dist
+
+    - name: Install the wheel for Windows configuration
+      if: ${{ runner.os == 'Windows' }}
+      run: |
+        $wheel = Get-ChildItem -Path dist -Filter "periodictable*.whl" | Select-Object -First 1
+        python -m pip install $wheel.FullName
+      shell: pwsh
+
+    - name: Install the wheel for configurations other than Windows
+      if: ${{ runner.os != 'Windows' }}
+      run: python -m pip install dist/periodictable*.whl
+
     - name: Install Python dependencies
       run: |
-        python -m pip install --upgrade pip
-        python -m pip install wheel setuptools
         python -m pip install numpy scipy matplotlib pytest pytest-cov
 
     - name: Test with pytest
       run: |
-        pytest -v
-
-    - name: check that the docs build (linux only)
-      if: matrix.cfg.doc == 1
-      run: |
-        python -m pip install sphinx
-        make -j 4 -C doc/sphinx SPHINXOPTS="-W --keep-going" html
+        pytest -v
```

Comment:
Some tests in package (e.g., `periodictable/fasta.py`) and others in test directory. There are doc tests in the guide and the api docs extracted from the package directory. Need to verify that these tests are run against the installed wheel rather than the source tree.